src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/directionissued/DirectionIssuedAboutToSubmitHandler.java

```diff
@@ -129,16 +244,25 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
             caseData.setTranslationWorkOutstanding("Yes");
             clearBasicTransientFields(caseData);
 
-        }
-        log.info("Saved the new interloc direction document for case id: " + caseData.getCcdCaseId());
 
+            if (caseDetails.getState().equals(State.INTERLOCUTORY_REVIEW_STATE)
+                    && caseData.getDirectionTypeDl() != null
+                    && StringUtils.equals(DirectionType.APPEAL_TO_PROCEED.toString(), caseData.getDirectionTypeDl().getValue().getCode())) {
+
+                PreSubmitCallbackResponse<SscsCaseData> response = serviceRequestExecutor.post(callback, bulkScanEndpoint);
+                sscsCaseDataPreSubmitCallbackResponse.addErrors(response.getErrors());
+
+            }
+            log.info("Saved the new interloc direction document for case id: " + caseData.getCcdCaseId());
+        }
         return sscsCaseDataPreSubmitCallbackResponse;
     }
 
-    private void updateCaseAfterExtensionRefused(SscsCaseData caseData, String interlocReviewState, State state) {
-        caseData.setHmctsDwpState("sentToDwp");
-        caseData.setDateSentToDwp(LocalDate.now().toString());
-        caseData.setInterlocReviewState(interlocReviewState);
-        caseData.setState(state);
+    private boolean shouldSetDwpState(SscsCaseData caseData) {
+        return ! reinstatementFeatureFlag
+                || isNull(caseData.getReinstatementOutcome())
+                || (!caseData.getReinstatementOutcome().equals(ReinstatementOutcome.GRANTED)
+                && !caseData.getReinstatementOutcome().equals(ReinstatementOutcome.REFUSED));
```

Comment:
(!caseData.getReinstatementOutcome().equals(ReinstatementOutcome.GRANTED) 
&& !caseData.getReinstatementOutcome().equals(ReinstatementOutcome.REFUSED))

would be good to change the method name to the scenarios where (or perhaps where not in this case) the DwpState should be set.